lib/Api.js

```diff
@@ -384,20 +391,21 @@ class Api {
                     }
                 });
             }
-
-            // libraries
             if (obj.libraries) {
+                const isSPM = isSwiftPackagePlugin(plugin);
                 Object.keys(obj.libraries).forEach(key => {
                     let podJson = Object.assign({}, obj.libraries[key]);
-                    podJson = replacePodSpecVariables(podJson, installOptions);
-                    const val = podsjsonFile.getLibrary(key);
-                    if (val) {
-                        events.emit('warn', `${plugin.id} depends on ${podJson.name}, which may conflict with another plugin. ${podJson.name}@${val.spec} is already installed and was not overwritten.`);
-                        podsjsonFile.incrementLibrary(key);
-                    } else {
-                        podJson.count = 1;
-                        podsjsonFile.setJsonLibrary(key, podJson);
-                        podfileFile.addSpec(podJson.name, podJson);
+                    if (!isSPM || (isSPM && _isTrue(podJson.nospm))) {
```

Comment:
I think this needs to be `!_isTrue(podJson.nospm)`?

* if `<platform name="ios">` and `<pod ...>`
  * **install**
* if `<platform name="ios" package="swift">` and `<pod ...>`:
  * **install**
* if `<platform name="ios" package="swift">` and `<pod nospm="false" ...>`:
  * **install**
* if `<platform name="ios" package="swift">` and `<pod nospm="true" ...>`:
  * _skip_